backend/firebase/firebase.tsx

```diff
@@ -9,14 +9,14 @@ import { FacebookAuthProvider, getAuth, GoogleAuthProvider, TwitterAuthProvider
 // Your web app's Firebase configuration
 // For Firebase JS SDK v7.20.0 and later, measurementId is optional
 const firebaseConfig = {
-    apiKey: "AIzaSyAWCKEkLBpYgN_dIJl6ncEHPngL21j3kz4",
-    authDomain: "okb-hope.firebaseapp.com",
-    projectId: "okb-hope",
-    storageBucket: "okb-hope.appspot.com",
-    messagingSenderId: "321787147976",
-    appId: "1:321787147976:web:faae136e03900bd47b97fd",
-    measurementId: "G-V7SV4XK6GC"
-  };
+  apiKey: "AIzaSyAWCKEkLBpYgN_dIJl6ncEHPngL21j3kz4",
+  authDomain: "okb-hope.firebaseapp.com",
+  projectId: "okb-hope",
+  storageBucket: "okb-hope.appspot.com",
+  messagingSenderId: "321787147976",
+  appId: "1:321787147976:web:faae136e03900bd47b97fd",
+  measurementId: "G-V7SV4XK6GC"
+};
```

Comment:
This should not be here

```suggestion
import config from .firebase.json
 const firebaseConfig = config
```